website/docs/r/monitor_diagnostic_setting.html.markdown

```diff
@@ -75,25 +74,27 @@ The following arguments are supported:
 
 * `log` - (Optional) One or more `log` blocks as defined below.
 
+-> **NOTE:** `log` has been superseded by `enabled_log` and will be removed in version 4.0 of the AzureRM Provider.
```

Comment:
We should have a deprecation on the property?

Reply:
changed